internal/config/config.go

```diff
@@ -237,3 +243,30 @@ func (c *Config) Keys(mount string) []string {
 
 	return nil
 }
+
+// DefaultLengthFromEnv will determine the password length from the env variable
+// GOPASS_PW_DEFAULT_LENGTH or fallback to the hard-coded default length.
+// If the env variable is set by the user and is valid, the boolean return value
+// will be true, otherwise it will be false.
+func DefaultLengthFromEnv(ctx context.Context) (int, bool) {
+	def := DefaultLength
+	cfg := FromContext(ctx)
+
+	if l := cfg.GetInt("generate.length"); l > 0 {
+		def = l
+	}
```

Comment:
You could use `config.Int(ctx, "generate.length")`

```suggestion
	if l := config.Int(ctx, "generate.length"); l > 0 {
		def = l
	}
```